### ISSUE TITLE
Enable more javascript

### ISSUE THREAD

**Issue by nonmakina** [towards-a-new-leftypol/leftypol_lainchan#65]
Which ones have we already added?
Which ones do we want?
What are their dependencies?

- [ ] ajax-post-controls.js
- [x] ajax.js
- [x] auto-reload.js
- [x] auto-scroll.js
- [ ] boardlist-alias.js
- [ ] boardlist-catalog-links.js
- [ ] catalog-link.js
- [x] catalog-search.js
- [x] catalog.js
- [ ] comment-toolbar.js
- [ ] compact-boardlist.js
- [ ] download-original.js
- [ ] expand-all-images.js
- [ ] expand-too-long.js
- [ ] expand-video.js
- [x] expand.js
- [ ] favorites.js
- [x] file-selector.js
- [ ] fireworks/fireworks.js
- [ ] fireworks/loadfireworks.js
- [ ] fireworks/soundmanager2-nodebug-jsmin.js
- [ ] fix-report-delete-submit.js
- [ ] forced-anon.js
- [ ] fullcalendar.min.js
- [x] gallery-view.js
- [ ] hearts.js
- [ ] hide-images.js
- [ ] hide-threads.js
- [ ] highlight.js
- [ ] highlight.pack.js
- [ ] hilight.js
- [ ] id_colors.js
- [ ] id_highlighter.js
- [ ] image-hover.js
- [ ] infinite-scroll.js
- [ ] inline-expanding-filename.js
- [ ] inline-expanding.js
- [ ] inline.js
- [ ] jquery-ui.custom.min.js
- [ ] jquery.min.js
- [ ] jquery.mixitup.min.js
- [ ] keyboard-shortcuts.js
- [ ] live-index.js
- [ ] loadballoons.js
- [ ] loadquill.js
- [ ] loadsceditor.js
- [ ] loadsnow.js
- [ ] loadsummernote.js
- [ ] loadtinymce.js
- [ ] local-time.js
- [ ] longtable/longtable.js
- [ ] mobile-default.js
- [ ] mobile-style.js
- [ ] mod/ban-list.js
- [ ] mod/recent-posts.js
- [ ] moment.min.js
- [ ] multi-image.js
- [ ] mute-images.js
- [ ] no-animated-gif.js
- [ ] options.js
- [ ] options/fav.js
- [ ] options/general.js
- [ ] options/user-css.js
- [ ] options/user-js.js
- [ ] own-board.js
- [ ] piwik.js
- [ ] post-filter.js
- [ ] post-hover.js
- [ ] post-menu.js
- [ ] prettify.js
- [ ] quick-post-controls.js
- [ ] quick-reply.js
- [ ] quill.js
- [ ] quote-selection.js
- [ ] rules-popup.js
- [ ] save-user_flag.js
- [ ] sceditor/development/jquery.sceditor.bbcode.js
- [ ] sceditor/development/jquery.sceditor.js
- [ ] sceditor/development/jquery.sceditor.xhtml.js
- [ ] sceditor/development/plugins/bbcode.js
- [ ] sceditor/development/plugins/format.js
- [ ] sceditor/development/plugins/undo.js
- [ ] sceditor/development/plugins/xhtml.js
- [ ] sceditor/languages/ar.js
- [ ] sceditor/languages/ca.js
- [ ] sceditor/languages/cn.js
- [ ] sceditor/languages/cs.js
- [ ] sceditor/languages/de.js
- [ ] sceditor/languages/el.js
- [ ] sceditor/languages/en-US.js
- [ ] sceditor/languages/en.js
- [ ] sceditor/languages/es.js
- [ ] sceditor/languages/et.js
- [ ] sceditor/languages/fa.js
- [ ] sceditor/languages/fr.js
- [ ] sceditor/languages/gl.js
- [ ] sceditor/languages/hu.js
- [ ] sceditor/languages/it.js
- [ ] sceditor/languages/ja.js
- [ ] sceditor/languages/lt.js
- [ ] sceditor/languages/nb.js
- [ ] sceditor/languages/nl.js
- [ ] sceditor/languages/pl.js
- [ ] sceditor/languages/pt-BR.js
- [ ] sceditor/languages/pt.js
- [ ] sceditor/languages/ru.js
- [ ] sceditor/languages/sv.js
- [ ] sceditor/languages/template.js
- [ ] sceditor/languages/tr.js
- [ ] sceditor/languages/tw.js
- [ ] sceditor/languages/uk.js
- [ ] sceditor/languages/vi.js
- [ ] sceditor/minified/jquery.sceditor.bbcode.min.js
- [ ] sceditor/minified/jquery.sceditor.min.js
- [ ] sceditor/minified/jquery.sceditor.xhtml.min.js
- [ ] sceditor/minified/plugins/bbcode.js
- [ ] sceditor/minified/plugins/format.js
- [ ] sceditor/minified/plugins/undo.js
- [ ] sceditor/minified/plugins/xhtml.js
- [ ] scheduler.min.js
- [ ] settings.js
- [ ] show-backlinks.js
- [ ] show-op.js
- [ ] show-own-posts.js
- [ ] smartphone-spoiler.js
- [ ] snowstorm.js
- [ ] strftime.min.js
- [ ] style-select.js
- [ ] summernote.js
- [ ] thread-stats.js
- [ ] thread-watcher.js
- [ ] threadscroll.js
- [ ] tinymce/jquery.tinymce.min.js
- [ ] tinymce/plugins/advlist/plugin.min.js
- [ ] tinymce/plugins/anchor/plugin.min.js
- [ ] tinymce/plugins/autolink/plugin.min.js
- [ ] tinymce/plugins/autoresize/plugin.min.js
- [ ] tinymce/plugins/autosave/plugin.min.js
- [ ] tinymce/plugins/bbcode/plugin.min.js
- [ ] tinymce/plugins/charmap/plugin.min.js
- [ ] tinymce/plugins/code/plugin.min.js
- [ ] tinymce/plugins/codesample/plugin.min.js
- [ ] tinymce/plugins/colorpicker/plugin.min.js
- [ ] tinymce/plugins/contextmenu/plugin.min.js
- [ ] tinymce/plugins/directionality/plugin.min.js
- [ ] tinymce/plugins/emoticons/plugin.min.js
- [ ] tinymce/plugins/fullpage/plugin.min.js
- [ ] tinymce/plugins/fullscreen/plugin.min.js
- [ ] tinymce/plugins/help/plugin.min.js
- [ ] tinymce/plugins/hr/plugin.min.js
- [ ] tinymce/plugins/image/plugin.min.js
- [ ] tinymce/plugins/imagetools/plugin.min.js
- [ ] tinymce/plugins/importcss/plugin.min.js
- [ ] tinymce/plugins/insertdatetime/plugin.min.js
- [ ] tinymce/plugins/legacyoutput/plugin.min.js
- [ ] tinymce/plugins/link/plugin.min.js
- [ ] tinymce/plugins/lists/plugin.min.js
- [ ] tinymce/plugins/media/plugin.min.js
- [ ] tinymce/plugins/nonbreaking/plugin.min.js
- [ ] tinymce/plugins/noneditable/plugin.min.js
- [ ] tinymce/plugins/pagebreak/plugin.min.js
- [ ] tinymce/plugins/paste/plugin.min.js
- [ ] tinymce/plugins/preview/plugin.min.js
- [ ] tinymce/plugins/print/plugin.min.js
- [ ] tinymce/plugins/save/plugin.min.js
- [ ] tinymce/plugins/searchreplace/plugin.min.js
- [ ] tinymce/plugins/spellchecker/plugin.min.js
- [ ] tinymce/plugins/tabfocus/plugin.min.js
- [ ] tinymce/plugins/table/plugin.min.js
- [ ] tinymce/plugins/template/plugin.min.js
- [ ] tinymce/plugins/textcolor/plugin.min.js
- [ ] tinymce/plugins/textpattern/plugin.min.js
- [ ] tinymce/plugins/toc/plugin.min.js
- [ ] tinymce/plugins/visualblocks/plugin.min.js
- [ ] tinymce/plugins/visualchars/plugin.min.js
- [ ] tinymce/plugins/wordcount/plugin.min.js
- [ ] tinymce/themes/inlite/theme.min.js
- [ ] tinymce/themes/modern/theme.min.js
- [ ] tinymce/tinymce.min.js
- [ ] titlebar-notifications.js
- [ ] toggle-images.js
- [ ] toggle-locked-threads.js
- [ ] treeview.js
- [ ] twemoji/twemoji.js
- [ ] upload-selection.js
- [ ] videojs-contrib-hls.js
- [ ] watch.js
- [ ] webm-settings.js
- [ ] webm/playersettings.js
- [ ] wpaint.js
- [ ] youtube.js

**Comment by nonmakina** [comment]
Marking this as high priority since it's low effort to implement, and high impact.

**Comment by nonmakina** [comment]
Lainchan is currently loading these scripts (not necessarily loading them in that order).
<img width="137" alt="image" src="https://user-images.githubusercontent.com/63084025/103335404-75368d00-4a3a-11eb-90c2-d25c4699372d.png">

**Comment by nonmakina** [comment]
We're currently loading these:

auto-reload.js
expand-video.js
hide-images.js
hide-threads.js
inline-expanding.js
jquery-ui.custom.min.js
jquery.min.js
local-time.js
multi-image.js
multi_image.js
options.js
post-hover.js
quick-reply.js
show-backlinks.js
show-op.js
style-select.js
webm-settings.js
youtube.js

**Comment by nonmakina** [comment]
This issue has expired. There's no new activity and we're searching for JS as we need it.
